contrib/clojure-package/src/org/apache/clojure_mxnet/gen/symbol.clj

```diff
@@ -0,0 +1,10940 @@
+(ns org.apache.clojure-mxnet.symbol
+    (:refer-clojure :exclude [* - + > >= < <= / cast concat identity flatten load max
+                              min repeat reverse set sort take to-array empty sin
+                              get apply shuffle])
+    (:require [org.apache.clojure-mxnet.util :as util])
+    (:import (org.apache.mxnet Symbol)))
+
+;; Do not edit - this is auto-generated
+
+;; Licensed to the Apache Software Foundation (ASF) under one or more
+;; contributor license agreements.  See the NOTICE file distributed with
+;; this work for additional information regarding copyright ownership.
+;; The ASF licenses this file to You under the Apache License, Version 2.0
+;; (the "License"); you may not use this file except in compliance with
+;; the License.  You may obtain a copy of the License at
+;;
+;;    http://www.apache.org/licenses/LICENSE-2.0
+;;
+;; Unless required by applicable law or agreed to in writing, software
+;; distributed under the License is distributed on an "AS IS" BASIS,
+;; WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+;; See the License for the specific language governing permissions and
+;; limitations under the License.
+;;
+
+
+
+
```

Comment:
same question as that in ndarray.

Reply:
It is the same process as described in the ndarray.